numpy/lib/tests/test_io.py

```diff
@@ -1835,6 +1835,13 @@ def test_auto_dtype_largeint(self):
         assert_equal(test['f1'], 17179869184)
         assert_equal(test['f2'], 1024)
 
+    def test_empty_file_with_converters(self):
+        with suppress_warnings() as sup:
+            sup.filter(message="genfromtxt: Empty input file:")
+            data = TextIO()
+            test = np.genfromtxt(data, converters={0: lambda arg: arg})
```

Comment:
Arggg I had `lambda arg: None` and the test was passing but now somehow something goes wrong ... I'll try to fix that tomorrow (European time).